examples/testplans/release.json

```diff
@@ -6,6 +6,9 @@
 	{"name": "Avocado source is sound",
 	 "description": "On your development machine, on a freshen Avocado source to be released, run `$ make check`. Expected result: Make command should say OK."},
 
+    {"name": "Avocado source does not contain spelling errors",
+     "description": "On your development machine, on a freshen Avocado source to be released, run `$ make spell`. Expected result: Make command should say OK."},
+
```

Comment:
Oh, softspace issue again (always forgot that `release.json` uses tabs and my vim automatically converts it to softtabs and screws it up. I'll send v2).